src/lib/profiles/data-management/Current/LoggingManagement.cpp

```diff
@@ -1783,8 +1699,7 @@ void LoggingManagement::FlushHandler(System::Layer * inSystemLayer, INET_ERROR i
 
     switch (mState)
     {
-    case kLoggingManagementState_Idle:
-    {
+    case kLoggingManagementState_Idle: {
```

Comment:
Style: new line before this line, consistent with line 1751.

Reply:
done